en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -1,118 +1,163 @@
+    |order                                         | The execution order in which the Data Source Health checker is executed. The default value is `97`.|
+    |monitored.datasources                         | This config is used to specify the Data Sources to be monitored as comma-separated values.|
+ 
 
-This is an open API which should ideally be blocked at the load balancer level. To invoke it, start the WSO2 product and send a GET request to the health check API. A sample cURL command is shown below.
+When you invoke the API and User Stores are healthy you can see a response as below:
 
-``` java
-    curl -k -v https://{hostname}:{port}/api/health-check/v1.0/health
 ```
-
-If the request is successful, you will recieve a 200 OK response (similar to the one shown below) with a list of health check results.
-
-``` java
-    {  
-       "health":[  
-          {  
+    {
+       "health":[
+          {
+             "key":"jdbc/SHARED_DB.connectivity.time.ms",
+             "value":"0"
+          },
+          {
+             "key":"WSO2MBStoreDB.connectivity.time.ms",
+             "value":"0"
+          },
+          {
+             "key":"jdbc/WSO2CarbonDB.connectivity.time.ms",
+             "value":"0"
+          },
+          {
+             "key":"jdbc/WSO2AM_DB.connectivity.time.ms",
+             "value":"0"
+          },
+          {
              "key":"jdbc/WSO2CarbonDB.active.connection.count",
              "value":"1"
           },
-          {  
-             "key":"bpsds.active.connection.count",
+          {
+             "key":"jdbc/WSO2MetricsDB.active.connection.count",
              "value":"1"
           },
-          {  
-             "key":"jdbc/WSO2MetricsDB.connectivityTime.ms",
-             "value":"81"
+          {
+             "key":"jdbc/WSO2MetricsDB.connectivity.time.ms",
+             "value":"33"
           },
-          {  
-             "key":"jdbc/WSO2MetricsDB.active.connection.count",
+          {
+             "key":"jdbc/WSO2AM_DB.active.connection.count",
              "value":"1"
           },
-          {  
-             "key":"jdbc/WSO2CarbonDB.connectivityTime.ms",
-             "value":"0"
+          {
+             "key":"jdbc/SHARED_DB.active.connection.count",
+             "value":"1"
           },
-          {  
-             "key":"bpsds.connectivityTime.ms",
-             "value":"0"
+          {
+             "key":"WSO2MBStoreDB.active.connection.count",
+             "value":"1"
           }
        ]
     }
 ```
 
 -   The `active.connection.count` parameter reflects the number of connections that are active.
 
--`ConnectivityTime` refers to the the duration of the connection.
+-   `ConnectivityTime` refers to the duration of the connection.
+
+#### User Store Health Checker
+
+This health checker returns the status of the user stores available in the system. You can configure the user stores to be monitored by adding the following configuration.
```

Comment:
can I select which user stories to be configured? if so how do I specify which user store? what is the value I should add?

Reply:
Yes you can configure the user stores to be configured.
```
    [carbon_health_check.health_checker.super_tenant_health_checker.properties]
    'monitored.user.stores' = "primary" 
```

This config is included in the latter part of the doc